pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs

```diff
@@ -0,0 +1,434 @@
+                </div>
+                <div class="modal-footer">
+                  <input type="hidden" name="__action" value="addGroup">
+                  <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                  <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                  <button type="submit" class="btn btn-primary">Add</button>
+                </div>
+              </form>
+            </div>
+          </div>
+        </div>
+        <div class="modal fade" id="configModal" tabindex="-1" role="dialog" aria-labelledby="configModalLabel">
+          <div class="modal-dialog" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h1 class="h4 modal-title" id="configModalLabel">Group Config</h1>
+              </div>
+              <form name="config-Group-form" method="POST">
+                <div class="modal-body">
+                  <span class="badge badge-secondary">id</span> <%= config_info.id %><br>
+                  <span class="badge badge-secondary">course_instance_id</span>
+                  <%= config_info.course_instance_id %><br>
+                  <span class="badge badge-secondary">minimum size</span> <%= config_info.minimum %><br>
+                  <span class="badge badge-secondary">maximum size</span> <%= config_info.maximum %><br>
+                  <span class="badge badge-secondary">Student Permissions</span> <%= config_info.permission %><br>
+                  <br>
+                  <div class="form-min">
+                    <label for="formmin">Minimum group size</label>
+                    <input type="text" value=<%= config_info.defaultmin %> class="form-control" id="formmin" name="minsize">
+                  </div>
+                  <div class="form-max">
+                    <label for="formman">Maximum group size</label>
+                    <input type="text" value=<%= config_info.defaultmax %> class="form-control" id="formmax" name="maxsize">
+                  </div>
+                  <label for="formman">Student Permissions</label> <br>
+                  <div class="form-check form-check-inline">
+                    <input class="form-check-input" type="checkbox" name="joincheck" value="true">
+                    <label class="form-check-label" for="inlineCheckbox1">Join Groups</label>
+                  </div>
+                  <div class="form-check form-check-inline">
+                    <input class="form-check-input" type="checkbox" name="createcheck" value="true">
+                    <label class="form-check-label" for="inlineCheckbox2">Create Groups</label>
+                  </div>
+                  <div class="form-check form-check-inline">
+                    <input class="form-check-input" type="checkbox" name="quitcheck" value="true">
+                    <label class="form-check-label" for="inlineCheckbox3">Quit Groups</label>
```

Comment:
I recognize this would require a large-ish refactor, but consider using "leave" instead of "quit", this is a better opposite for "join".